pkg/monitoring/cdi-controller/metrics/dataimportcron_metrics.go

```diff
@@ -0,0 +1,37 @@
+package metrics
```

Comment:
@avlitman Please move all the metrics so that they will be under /pkg/monitoring/metrics/.
Please see what it will mean to have both the cdi-controller metrics and the operator-controller metrics files flattened with a prefix like:
 /pkg/monitoring/metrics/cdi_controller_dataimportcron.go
 /pkg/monitoring/metrics/cdi_controller_storageprofile.go
 /pkg/monitoring/metrics/operator_controller.go